mycroft/tts/say_tts.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import subprocess
+
+from mycroft.tts import TTS, TTSValidator
+
+
+class Say(TTS):
```

Comment:
Since say is pretty vague in the context of text to speech (even though that's what it's called), would you mind renaming it to something like `RSynth`? This also gives people a better search term to use to download it.